<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissingWadsDialog</class>
 <widget class="QDialog" name="MissingWadsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doomseeker - files are missing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="areaWadseekerAlreadyRunning" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>You don't have all the files required by this server and an instance of Wadseeker is already running.

Press 'Ignore' to join anyway.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="areaWadseekerCanBeRun" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="areaCantBeDownloaded" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>These files belong to a commercial game or are otherwise blocked from download:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCantBeDownloadedFiles">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;files&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Make sure that this file is in one of the paths specified in Options -&gt; File Paths.

If you don't have this file, and it belongs to a commercial game, you need to purchase the game associated with this file. Wadseeker will not download commercial IWADs or modifications.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblInstallFreedoom">
           <property name="text">
            <string>You can also install a free replacement IWAD with &quot;Install Freedoom&quot; button.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="areaCanBeDownloadedFiles" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Following files can be downloaded:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCanBeDownloadedFiles">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;files&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="areaOptionalFiles" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Optional files:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="optionalFilesList"/>
         </item>
         <item>
          <widget class="Line" name="line_6">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Do you want Wadseeker to find the missing WADs?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblUseIgnore">
        <property name="text">
         <string>Alternatively use ignore to connect anyways.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnInstallFreedoom">
        <property name="text">
         <string>Install Freedoom</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnInstall">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Install</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnIgnore">
        <property name="text">
         <string>Ignore</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>MissingWadsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>703</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnIgnore</sender>
   <signal>clicked()</signal>
   <receiver>MissingWadsDialog</receiver>
   <slot>ignoreMissingFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>703</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnInstall</sender>
   <signal>clicked()</signal>
   <receiver>MissingWadsDialog</receiver>
   <slot>installMissingFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>703</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnInstallFreedoom</sender>
   <signal>clicked()</signal>
   <receiver>MissingWadsDialog</receiver>
   <slot>installFreedoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>703</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optionalFilesList</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>MissingWadsDialog</receiver>
   <slot>updateStateAccordingToFileSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>668</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>installFreedoom()</slot>
  <slot>installMissingFiles()</slot>
  <slot>ignoreMissingFiles()</slot>
  <slot>updateStateAccordingToFileSelection()</slot>
 </slots>
</ui>
